src/main/java/org/sonarqube/gradle/AndroidUtils.java

```diff
@@ -101,6 +108,30 @@ private static void configureForAndroid(Project project, BaseVariant variant, Ma
     }
   }
 
+  private static void populateSonarQubeAndroidProperties(AndroidVariantAndExtension android, Map<String, Object> properties) {
```

Comment:
I propose populateSonarQubeSdkVersionAndroidProperties  since we are populating android props elsewhere too